content/articles/2019-02-bigrquery-1-1-0.Rmarkdown

```diff
@@ -0,0 +1,40 @@
+---
+title: bigrquery 1.1.0
+author: Mara Averick
+date: '2019-02-15'
+slug: bigrquery-1-1-0
+description: > 
+  bigrquery 1.1.0 is now on CRAN.
+categories:
+  - package
+photo:
+  url: https://www.pexels.com/photo/green-and-red-abstract-painting-1070534/
+  author: Steve Johnson
+---
+
+```{r setup, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE, 
+  comment = "#>"
+)
+```
+
+We're delighted to announce that [bigrquery](https://bigrquery.r-dbi.org/) 1.1.0 is now on CRAN. bigrquery makes it easy to work with data stored in [Google BigQuery](https://developers.google.com/bigquery/), a hosted database for big data. This is a minor release, with some improved type support, and SQL translation. For a complete list of updates, please see the [Changelog](https://bigrquery.r-dbi.org/news/index.html).
+
+## Improved type support
+
+[`bq_table_download()`](https://bigrquery.r-dbi.org/reference/bq_table_download.html) and the [`DBI::dbConnect`](https://www.rdocumentation.org/packages/DBI/versions/0.5-1/topics/dbConnect) method now have a `bigint` argument, which governs how BigQuery integer columns are imported into R. As before, the default will be `bigint = "integer"`. However, you can now set `bigint = "integer64"` to import BigQuery integer columns as `bit64::integer64` columns in R. This allows for values outside the range of integer (`-2147483647` to `2147483647`).
```

Comment:
Is the `bigint` argument actually new, or just the `bigint = "integer64"` option? Also, I took the last sentence from NEWS, but those numbers are the bounds for what's aloud when `bigint = "integer"` is set, right? If so, I might reword to clarify.

Reply:
The argument is new — can you please credit Rasmus for this, because he basically did all the work for it.